airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/datatypes.sql

```diff
@@ -69,19 +77,25 @@
 {% endmacro %}
 
 
-{# numeric ------------------------------------------------- #}
+{#-- numeric ------------------------------------------------- --#}
```

Comment:
Why are you adding the extra two `--` at the start and end of comment block btw?

Reply:
Added those, for better reading of comments, sometimes you lost reading Jinja macros) If you insist - i'll clean it up.

Reply:
it's just consistent with all the comments in our macros... if you find it more readable, can you apply it to all the other comments too? (search/replace)

Reply:
I'll revert these changes, for consistency)